src/libraries/Microsoft.PowerFx.Core/Public/Types/FormulaType.cs

```diff
@@ -148,7 +151,7 @@ internal static FormulaType Build(DType type)
                     return BindingError;
 
                 default:
-                    throw new NotImplementedException($"Not implemented type: {type}");
+                    throw new NotImplementedException($"Not implemented type: {type} ({type.Kind}) ({type?.GetType().FullName})");
```

Comment:
TODO: revert this change. Was useful for debugging.

Reply:
I'd approve one this change is reverted.

Reply:
Note that this is an outdated change. All of this is replaced with unsupported type construction.